### DISCUSSION TITLE
DataTemplateSelector in TreeView

### DISCUSSION THREAD

**Discussion by KOE73** [enisn/UraniumUI#475] [Ideas]
I tried to implement a DataTemplateSelector for TreeView and it didn't work like here https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/datatemplate

It would be a good idea for the DataTemplateSelector to be used in the TreeView.

**Comment by enisn** [comment]
I need to check what is the problem, it should work by default, I didn't implement anything different than default DataTemplates.

I'm creating an issue for it. Thanks
